.github/workflows/linux_conda.yml

```diff
@@ -0,0 +1,83 @@
+name: 'linux / conda'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
+  cancel-in-progress: true
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  py310:
+    runs-on: ubuntu-20.04
+    name: 'linux conda 3.10'
+    defaults:
+      run:
+        shell: bash
+    env:
+      CONDA_ENV: 'environment.yml'
+      DISPLAY: ':99.0'
+      MNE_LOGGING_LEVEL: 'warning'
+      MKL_NUM_THREADS: '1'
+      PYTHONUNBUFFERED: '1'
+      PYTHON_VERSION: '3.10'
+    steps:
+      - name: 'checkout MNE-Python repo'
+        uses: actions/checkout@v3
+        with:
+          name: mne-tools/mne-python
+          ref: refs/heads/main
```

Comment:
for easier review: this step has changed from how it was in the MNE-Python repo: it specifically clones mne-python instead of mne-incubator, so we don't have to copy over all the `./tools/*.sh` scripts.